protocol/requests/create_community_request.go

```diff
@@ -19,6 +19,8 @@ var (
 type CreateCommunity struct {
 	Name                         string                               `json:"name"`
 	Description                  string                               `json:"description"`
+	IntroMessage                 string                               `json:"introMessage"`
+	OutroMessage                 string                               `json:"outroMessage"`
```

Comment:
Probably makes sense to have these `omitempty` to ensure this won't break when clients create `CreateCommunity` requests that don't include these fields